x-pack/plugins/cloud_security_posture/public/common/api/use_license_management_locator_api.tsx

```diff
@@ -5,23 +5,21 @@
  * 2.0.
  */
 
-import { useQuery } from '@tanstack/react-query';
 import { useKibana } from '../hooks/use_kibana';
 
 const LICENSE_MANAGEMENT_LOCATOR = 'LICENSE_MANAGEMENT_LOCATOR';
 
-const getLicenseManagementLocatorKey = 'license_management_url_key';
-
+/**
+ * Hook to get the license management locator
+ * @returns a callback to navigate to the license management page
+ */
 export const useLicenseManagementLocatorApi = () => {
   const { share } = useKibana().services;
 
-  return useQuery([getLicenseManagementLocatorKey], () => {
-    const locator = share.url.locators.get(LICENSE_MANAGEMENT_LOCATOR);
-    // license management does not exist on serverless
-    if (!locator) return;
+  const locator = share.url.locators.get(LICENSE_MANAGEMENT_LOCATOR);
```

Comment:
While working on how to mock the API call for this method, realized that it doesn't perform an API call, so I removed the `useQuery` as it's not necessary.